src/EventPage.js

```diff
@@ -25,6 +26,12 @@ export default function EventPage() {
     onPacklistSave,
     goPacklistBack,
   } = usePacklistForm(addPacklist)
+  const hasHiddenEvent = eventArray.some((event) => event.isHide === true)
```

Comment:
`isHidden` would match the other occurances of this word better

```suggestion
  const hasHiddenEvent = eventArray.some((event) => event.isHidden === true)
```

Reply:
done

Reply:
Check for strict equality with `true` is not necessary, make it short and sweet `events.some(event => event.isHidden)`